membership/src/main/scala/zio/membership/hyparview/periodic.scala

```diff
@@ -9,48 +9,43 @@ import zio.logging.Logging.Logging
 object periodic {
 
   def doShuffle[T: Tagged]: ZIO[Views[T] with Logging with HyParViewConfig with TRandom, Nothing, ViewState] =
-    Views.using[T].apply { views =>
-      TRandom.using { tRandom =>
-        getConfig.flatMap { config =>
-          val go: IO[SendError, ViewState] =
-            views.activeView
-              .map(_.toList)
-              .flatMap { nodes =>
-                tRandom.selectOne(nodes).flatMap {
-                  case None => STM.succeed(views.viewState.commit)
-                  case Some(node) =>
-                    for {
-                      active  <- tRandom.selectN(nodes.filter(_ != node), config.shuffleNActive)
-                      passive <- views.passiveView.flatMap(p => tRandom.selectN(p.toList, config.shuffleNPassive))
-                      state   <- views.viewState
-                    } yield views
-                      .send(
-                        node,
-                        ActiveProtocol
-                          .Shuffle(views.myself, views.myself, active, passive, TimeToLive(config.shuffleTTL))
-                      )
-                      .as(state)
-                }
-              }
-              .commit
-              .flatten
-          go.eventually
-        }
-      }
+    getConfig.flatMap { config =>
+      val go: ZIO[TRandom with Views[T], SendError, ViewState] =
+        Views.activeView
+          .map(_.toList)
+          .flatMap { nodes =>
+            TRandom.selectOne(nodes).flatMap {
+              case None => STM.succeed(Views.viewState.commit)
```

Comment:
Why not `Views.viewState` only?

Reply:
It expects an STM[_, UIO[_]], so I rolled with this one